core/libraries/plugin_api/EE_Register_Privacy_Policy.lib.php

```diff
@@ -0,0 +1,75 @@
+<?php
+defined('EVENT_ESPRESSO_VERSION') || exit('No direct script access allowed');
+
+
+
+/**
+ * Class EE_Register_Privacy_Policy
+ * Registers additional plugin privacy policy text. Yes, you could manually call `wp_add_privacy_policy_content`
+ * but you'd also need to add the logic about when to load the code etc. This way EE core takes care of knowing
+ * when to load this class and call it
+ *
+ * @package        Event Espresso
+ * @author         Mike Nelson
+ * @since          $VID:$
+ */
+class EE_Register_Privacy_Policy implements EEI_Plugin_API
+{
+
+    /**
+     * FQCN for all privacy policy generators
+     *
+     * @var array keys are plugin_ids, and values are an array of FQCNs or FQCNs
+     */
+    protected static $privacy_policies = array();
+
+
+
+    /*
+     * @param string $plugin_id
+     * @param array $FQNSs can be the fully qualified namespaces each containing only privacy policies,
+     *              OR fully qualified class names of privacy policies
+     */
+    public static function register($plugin_id = null, $FQCNs = null)
```

Comment:
sry missed this too, but the parameter defaults should match their data type, so strings should default to `''`, arrays should default to `array()`, and `null` should be used as a default for optional objects.

As well, we should type hint for array as much as possible, so in other words, this method signature should change to:

```public static function register($plugin_id = '', array $FQCNs = array())```

one lasts thing... do we even want to have either of these as optional?

Reply:
kk, ya `$FQCNs` should type-hint for array. 
The interface dictates they need to be optional though

Reply:
oh, actually the interface dictates it can't be type-hinted for, actually. So I just changed it to default to an array